### ISSUE TITLE
Make ODK form entry available on browsers

### ISSUE THREAD

**Issue by ethan92429** [AguaClara/POST#23]
@jacuerp we just tried to put ODK on the phone at Atima, but their internet only works on a desktop computer, so we need a way for them to be able to fill the form out via some browser method. This is not fully supported in the ODK software, but there are other companies that provide this service and interface with ODK. Enketo seems like the best option. I think we can somehow plug it in and it uses the already existing xls odk form to create the same questions and I think it then sends that data directly to aggregate.

**Comment by ethan92429** [comment]
Just researched the alternative webforms. ODK recommends Enketo, touchforms or Purcforms. However, purcforms is outdated, Enketo costs money and touchforms are meant for touchscreens and require setting up a Django server. I also found [this post](https://groups.google.com/forum/#!topic/opendatakit/tMPolBlFnDw) suggesting openxdata, but that appears to be broken/costs money. An alternative is to set up a corresponding google form for each defined ODK collect form. The Google form responses can then be put directly into the Fusion Table using an apps script.

**Comment by ethan92429** [comment]
I implemented this approach, and the corresponding google form can be found in the drive under web forms. However, we made the decision to focus on only the smartphone app so that we won't have to keep updating the corresponding web form. Once there is demonstrated greater demand for the browser form option, we can focus on coming up with a more stable/sustainable methodology for constructing the browser forms.
